### ISSUE TITLE
function.js

### ISSUE THREAD

**Issue by phynias** [loganmarchione/terraform-aws-static-site#3]
a reference to function.js file is made but the file isn't in the repo.

**Comment by loganmarchione** [comment]
Nice catch! It's in my repo where I call the module but I forgot to put a sample function in the module itself. I'll correct this tonight or tomorrow. Thanks!

**Comment by phynias** [comment]
here is one i am using, it makes it so sub dirs can be called without index.html

```
function handler(event) {
    var request = event.request;
    var uri = request.uri;
    if (uri.endsWith('/')) {
        request.uri += 'index.html';
    }
    else if (!uri.includes('.')) {
        request.uri += '/index.html';
    }
    return request;
}
```

**Comment by loganmarchione** [comment]
Yep that's the same one I was using. I just released `0.1.1`  with that fix. Thanks for the catch!

https://github.com/loganmarchione/terraform-aws-static-site/tags
